dom.go

```diff
@@ -243,6 +244,12 @@ func (h *HTML) reconcileProperties(prev *HTML) {
 		}
 	}
 
+	// Classes
+	classList := h.node.Get("classList")
+	for name := range h.classes {
+		classList.Call("add", name)
```

Comment:
It may be good to assume the DOM API is slow than a map check, i.e. before calling `add` check `prev`:

```
if _, ok := prev.classes[name]; !ok {
    classList.Call("add", name)
}
```

But this is just a hunch / best practice in my mind. It's probably plenty fast so I am OK with this as-is if you don't wish to update.

Reply:
It's probably plenty fast, but I'm fine with the change as I haven't benchmarked.